osu.Game/Overlays/News/Sidebar/YearsPanel.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Shapes;
+using osuTK;
+using System.Linq;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using System.Collections.Generic;
+using osu.Game.Graphics;
+using osu.Framework.Bindables;
+using System.Collections.Specialized;
+
+namespace osu.Game.Overlays.News.Sidebar
+{
+    public class YearsPanel : CompositeDrawable
+    {
+        public int[] Years
+        {
+            set
+            {
+                years.Clear();
+                years.AddRange(value);
+            }
+        }
+
+        private readonly BindableList<int> years = new BindableList<int>();
```

Comment:
would like to see this gone. possible direction how that can be achieved has been outlined above (caching/resolving bindables), or just publically exposing the bindable list. if you don't care about the nice value change callbacks you could also ultimately do `Bindable<int[]>`, too, i suppose.

Reply:
Or I just can place everything in the ctor so just property will do. But will cache metadata anyway.

Reply:
Nvm, thare will be no need in property then at all